sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/PruneFileSourcePartitionsSuite.scala

```diff
@@ -108,4 +108,10 @@ class PruneFileSourcePartitionsSuite extends QueryTest with SQLTestUtils with Te
       }
     }
   }
+
+  override def getScanExecPartitionSize(query: String): Long = {
```

Comment:
nit: the input can be spark plan instead of query string.

Reply:
> nit: the input can be spark plan instead of query string.

Updated